### ISSUE TITLE
Updating base class error

### ISSUE THREAD

**Issue by Cuel** [fparma/fp_factions#1]
```
23:30:22 Updating base class CAManBase->Man, by FP_Factions\gsfg\config.bin/cfgvehicles/SoldierEB/
```

**Comment by Tinter** [comment]
The update is only

**Comment by Tinter** [comment]
The update is only
	class SoldierEB: Man{};

**Comment by Tinter** [comment]
The update is only 
	class SoldierEB: Man{}; 
but could really be just 
	class SoldierEB; 

	class Man;
 is useless as nothing is inherited from it.

**Comment by Mallow234** [comment]
This has since been fixed.

**Comment by Cuel** [comment]
you can auto close issues by writing it in the commit 
e.g
Fixed UBC error, fixes #1
